adcc/ExcitedStates.py

```diff
@@ -216,40 +216,37 @@ def __init__(self, data, method=None, property_method=None,
             if hasattr(data, "eigenvectors"):
                 self.excitation_vectors = data.eigenvectors
             if hasattr(data, "excitation_energies"):
-                self.__excitation_energies = data.excitation_energies
+                # TODO: what if data has energy corrections?
+                # make better ctor if data is other ExcitedStates object
```

Comment:
That's a good point and definitely happens for CIS for example. I think this should be handled properly.

Reply:
Handled.